components/apimgt/org.wso2.carbon.apimgt.impl/src/main/java/org/wso2/carbon/apimgt/impl/monetization/DefaultMonetizationImpl.java

```diff
@@ -69,4 +78,28 @@ public boolean disableMonetization(String tenantDomain, API api, Map<String, Str
     public Map<String, String> getTotalRevenue(API api, APIProvider apiProvider) throws MonetizationException {
         return new HashMap<String, String>();
     }
+
+    @Override
+    public boolean publishMonetizationUsageRecords(MonetizationUsagePublishInfo monetizationUsagePublishInfo)
```

Comment:
shall we add method level Java doc comments?

Reply:
Added in https://github.com/wso2/carbon-apimgt/pull/7953/commits/467a00b5a076000a5b86b18f024212566acc7868